### PR TITLE
Improve coverage chsdi/views/wmtscapabilities.py

### PR DESCRIPTION
**Pull request by mariavkourouni**

### CODE REVIEW

File: chsdi/views/wmtscapabilities.py [hunk 1]
```diff
@@ -84,7 +84,7 @@ def wmtscapabilities(self):
         if hasattr(self.models['GetCapThemes'], 'oberthema_id'):
             themes = self.request.db.query(self.models['GetCapThemes']).order_by(self.models['GetCapThemes'].oberthema_id).all()
         else:
-            themes = self.request.db.query(self.models['GetCapThemes']).all()
+            themes = self.request.db.query(self.models['GetCapThemes']).all()  # pragma: no cover
```
Comment (loicgasser) [inline chsdi/views/wmtscapabilities.py]: this is potentially dead code maybe better to just remove it if oberthema_id is always an attribute, please double check

**Comment by loicgasser** [comment]
squash please.

**Comment by gjn** [comment]
@procrastinatio Can you verify that this is never used?

**Comment by gjn** [comment]
@mariavkourouni I'll close this. Better try a way to test this code.

**Comment by mariavkourouni** [comment]
@gjn yes I will

**Pull request closed without merging**